### DISCUSSION TITLE
If it's possible, I think a lasso tool for selection would be very beneficial

### DISCUSSION THREAD

**Discussion by carolinaqzx** [zsviczian/obsidian-excalidraw-plugin#645] [Ideas]
I am starting to use Excalidraw in Obsidian as a bullet journal (thank you Zsolt!) so I usually hand write my notes, rather than type them. I often have to move around text or images that I draw, but I have a difficult time selecting them properly, especially if the strokes of one word are too close to another word. I would really appreciate the incorporation of a lasso tool for more accurate selections; however, I am not even sure that you have the ability to do that (perhaps I should make this suggestion to the creators of Excalidraw instead!)

**Comment by YesterKo** [comment]
Yep, you should post it to the Excalidraw team instead, as Zsviczian tries to keep modifications to the custom Excalidraw engine minimal.

**Reply by jludvice** [reply]
I believe this is relevant discussion in excalidraw project in case anyone finds this discussion
https://github.com/excalidraw/excalidraw/issues/6350
